python/ray/tests/test_gcs_fault_tolerance.py

```diff
@@ -815,7 +815,7 @@ def pid(self):
     cluster.head_node.kill_gcs_server(False)
 
     print("Start gcs")
-    sleep(2)
```

Comment:
We should update the whole test: with this PR, gcs shouldn't exit during redis leader failover but just reconnect